src/main/java/com/google/devtools/build/docgen/templates/attributes/common/tags.html

```diff
@@ -28,13 +28,21 @@
   </li>
 
   <li><code>no-cache</code> keyword results in the action or test never being
-    cached.
+    cached (remotely or locally)
   </li>
 
-  <li><code>no-remote</code> keyword results in the action or test never being
+  <li><code>no-remote-cache</code> keyword results in the action or test never being
+    cached remotely (but it may be cached locally; it may also be executed remotely).
+  </li>
+
+  <li><code>no-remote-exec</code> keyword results in the action or test never being
     executed remotely (but it may be cached remotely).
   </li>
 
+  <li><code>no-remote</code> keyword prevents the action or test from being executed remotely
+    or cached remotely. This is equivalent to using both no-remote-cache and no-remote-exec.
+  </li>
+
```

Comment:
Great, thanks a lot for updating the docs!
Could we maybe add a note that when we refer to local cache - we usually mean disk_cache, when to remote - we usually refer to http/grpc cache?